docker-compose.yml

```diff
@@ -5,41 +5,65 @@ volumes:
     driver: local
 
 services:
- postgres:
-   restart: always
-   image: postgres:latest
-   environment:
-     POSTGRES_USER: openoversight
-     POSTGRES_PASSWORD: terriblepassword
-     POSTGRES_DB: openoversight-dev
-   volumes:
-     - postgres:/var/lib/postgresql/data
-   ports:
-     - "5432:5432"
+  postgres:
+    restart: always
+    image: postgres:latest
+    environment:
+      POSTGRES_USER: openoversight
+      POSTGRES_PASSWORD: terriblepassword
+      POSTGRES_DB: openoversight-dev
+    volumes:
+      - postgres:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+    network_mode: "host"
```

Comment:
Why do we need to change all the containers to `network_mode: "host"`?

Reply:
Because we refer to the other docker services from both the frontend (unsigned url image lookups in HTML) and the backend (image uploading) - can't hit `http://minio:1234` from the browser, and it's not exposed in a format the browser can hit otherwise. The alternative that I was considering was making a proxy route from the backend - but I think that'd introduce more "are we in testing?" conditionals in the codebase that I was trying to avoid.

Reply:
Thanks for the explanation. I see you already mentioned that in the PR summary and I missed it, sorry.
So basically we only save one string for the URL of an image, but the backend would need a different host to connect to minio than the front-end. That makes sense!
The docker-documentation says "The host networking driver only works on Linux hosts, and is not supported on Docker Desktop for Mac, Docker Desktop for Windows, or Docker EE for Windows Server." Would this mean that with those changes, the dev-environment is only functional on Linux? In that case, I think we should definitely explore other options

Reply:
Oh! I never knew that! That sucks! I code on linux. I'll look into the proxy endpoint if that's okay? The image can be `http://localhost/proxy/file123.png` or something? and the backend can fetch it from S3?

Reply:
> Oh! I never knew that! That sucks! I code on linux.

I am also on Linux, so I cannot confirm that it doesn't work, but that's what I read from the documentation.

> I'll look into the proxy endpoint if that's okay? The image can be `http://localhost/proxy/file123.png` or something? and the backend can fetch it from S3/minio?

Sounds good to me. Thank you for investigating this!